analysis_options.yaml

```diff
@@ -1,11 +1,9 @@
 include: package:pedantic/analysis_options.1.9.0.yaml
 # https://dart-lang.github.io/linter/lints/
 analyzer:
-  exclude: 
+  exclude:
     - example/**
     - build/**
-    - lib/utils/logging.dart
-    - lib/utils/type.dart
```

Comment:
I've removed these because they were useful for catching errors. I moved the info-related analysis recommendations to the files themselves.